packages/tupaia-web/src/features/Dashboard/ExportDashboard/ExportDashboard.tsx

```diff
@@ -87,6 +87,9 @@ export const ExportDashboard = () => {
           exportWithLabels: false,
           exportWithTable: true,
           exportWithTableDisabled: false,
+          exportDescription: null,
+          exportDescriptionCharLimit: 250,
+          exportDescriptionCharLimitReached: false,
```

Comment:
These two can probably be contained within the input component, since it's just frontend validation. Things that need to go in the context are only things that are used across different components